feasto_core_clean_arch/models/item.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from . import restaurant
+from .restaurant import Restaurant
+
+
+class Item(models.Model):
+    name = models.CharField(max_length=50)
+    available_quantity = models.IntegerField()
+    restaurant = models.ForeignKey(Restaurant, on_delete=models.CASCADE,
```

Comment:
why related name is `restaurant`, isn't it the same as field name ? what are the reasons here ?